components/EmotionRecordsList.js

```diff
@@ -52,10 +82,33 @@ export default function EmotionRecordsList({ emotionEntries }) {
         }) => {
           return (
             <>
-              <StyledListItem key={id} onClick={() => handleShow(id)}>
-                {date}
-              </StyledListItem>
-              <StyledDetails $show={show[id]}>
+              <StyledListItemWrapper>
+                <StyledListItem key={id} onClick={() => handleShowDetails(id)}>
+                  {date}
+                </StyledListItem>
```

Comment:
You're using the key prop inside the map function on the `StyledListItem`, but since you have a wrapper around the list item it's not the first child. Best practice is to assign the key directly to the mapped component, which is here the `StyledListItemWrapper`:

```suggestion
              <StyledListItemWrapper key={id}>
                <StyledListItem onClick={() => handleShowDetails(id)}>
                  {date}
                </StyledListItem>
```


But: I noticed, that you use the `StyledListItemWrapper` only for `position: relative;`. You could remove the wrapper (and than also the styled component) by placing `position: relative;` directly on the `StyledListItem`. Than you have to adjust your code, by putting the button inside the list item:

```
<StyledListItem key={id} onClick={() => handleShowDetails(id)}>
                {date}
                <StyledDeleteButton
                  type="button"
                  aria-label="Delete Emotion Entry"
                  onClick={() => {
                    handleShowConfirmMessage(id);
                  }}
                />
</StyledListItem>
```

Reply:
Thanks for your review! The renaming is definitely much better. 

About the last suggestion. I didn't put the `button` inside the `ListItem` because clicking the `button` would also trigger the `detailView` to open which i would like to avoid. --> I changed the `key` to be on the `wrapper` instead!